internal/controller/node_reconciler.go

```diff
@@ -61,6 +58,7 @@ func (r *NodeReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 	}
 
 	podchanged := false
+	minCooldown := time.Duration(math.MaxInt64)
```

Comment:
but should this be default? default should be what k8s follow? (5 sec)?

Reply:
This will still get overridden by the 1 minute default in GetCooldown() if none is defiend. If no pdbwatchers are foudn we won't use this cooldown.